BUGPROCESS.md

```diff
@@ -0,0 +1,74 @@
+# Terraform Core GitHub Bug Triage & Labeling
+The Terraform Core team has adopted a more structured bug triage process than we previously used. Our goal is to respond to reports of issues quickly.
+
+When a bug report is filed, our goal is to either:
+1. Get it to a state where it is ready for engineering to fix it in an upcoming Terraform release, or 
+2. Close it explain why, if we can't help
```

Comment:
As a Terraform user, I greatly appreciate this attempt to reduce the number of open tickets :-)

I would also suggest to consider a method to close the stale/old tickets, not only the new ones.

Reply:
Agreed! I am just prioritizing the new issues as a triage step, and then we can work on the backlog of stale/old issues once we've got a good way to deal with the stream of incoming new ones.